docs/app/getting-started.md

```diff
@@ -21,7 +21,17 @@ A very simple [docker-compose.yml](/docker-compose.yml) has been included to sup
 
 3. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
 
-4. You'll also need [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+4. You'll also need a container runtime ([Colima](https://github.com/abiosoft/colima) and Docker CLI. You can install both of these with:
+
+   ```bash
+   brew install colima docker
+   ```
+
+   After you finish installing, you'll want to start the docker daemon with
+
+   ```bash
+   colima start
```

Comment:
My lack of Docker expertise is coming out here, but I don't really understand much of what you're saying (not sure what a docker context is for example), and I'm not sure what params to provide colima or whether those params should be defaulted across the organization or whether we should just provide a reference to colima docs for people to tune to their own needs and computer resources.

I haven't noticed a big performance penalty so far, and from the online threads I've come across some even claim it's faster than docker desktop's engine.

Reply:
If there are some learnings that you have that you want to add, do you want to jump on this branch and add those things? Otherwise I'm not really sure what additional guidance to give. I'm just trying to meet the MVP needs of getting people off Docker Desktop

Reply:
I'm actually running into some issues with colima. For some reason the image I built with `docker build` doesn't run in the cloud with the template-infra repo. The one with docker buildx build works. So I'm going to close this for now